### PR TITLE
print warning if no license file found in the package

### PR DESCRIPTION
**Pull request by AkihiroSuda**
cc @dnephin

Signed-off-by: Akihiro Suda <suda.akihiro@lab.ntt.co.jp>

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -85,6 +86,27 @@ func checkUnused(deps []depEntry, vd string) {
...
+		dir := filepath.Join(vd, d.importPath)
+		files, err := ioutil.ReadDir(dir)
+		if err != nil {
+			// err can be nil for unused package
+			continue
+		}
+		licenseFiles := 0
+		for _, file := range files {
+			p := filepath.Join(dir, file.Name())
+			if isLicenseFile(p) {
```
Comment (dnephin) [inline main.go]: This will match `README`, which I think is often not a license.

I think we need to fix `isLicenseFile()`

File: main.go [hunk 2]
```diff
@@ -85,6 +86,27 @@ func checkUnused(deps []depEntry, vd string) {
...
+				licenseFiles++
+			}
+		}
+		if licenseFiles == 0 && verbose {
+			log.Printf("WARNING(verbose): package %s may lack license information", d.importPath)
```
Comment (dnephin) [inline main.go]: I guess this will depend on the other branch so that it exits with a non-zero code?

File: main.go (hunk 1 above)
Comment (AkihiroSuda) [inline main.go]: Detecting README is intentional (https://github.com/LK4D4/vndr/pull/21), but maybe it is good to `grep -i license README*` in another PR

File: main.go (hunk 2 above)
Comment (AkihiroSuda) [inline main.go]: This should exits with zero, because we are not free from false-positive

Comment (dnephin) [inline main.go]: Oh, I didn't notice that this was only a warning in the old version. Thanks

**Comment by AkihiroSuda** [comment]
rebased

**Comment by LK4D4** [comment]
@AkihiroSuda needs test :)

**Comment by AkihiroSuda** [comment]
updated
sorry for delay

**Pull request merged**